cg/models/orders/json_sample.py

```diff
@@ -12,7 +12,7 @@ class JsonSample(OrderSample):
     index: Optional[str]
     quantity: Optional[str]
     synopsis: Optional[List[str]]
-    well_position: Optional[constr(regex="[A-H]:[0-9]+")]
+    well_position: Optional[constr(regex=r"[A-H]:[0-9]+")]
```

Comment:
have yo tested this? Flake thinks its a bug

Reply:
it can be tested more. I was hoping you would know what the error message implied 😇  I briefly saw that there are stackoverflow-posts about this so I guess there's a solution out there

Reply:
I.e I'll look into it 👍

Reply:
The problem is described here: https://github.com/PyCQA/pyflakes/issues/542, I'm not sure what the solution is but I don't want this PR to be blocked by it touching this line so I'll revert it